egs/aidatatang_200zh/s5/local/nnet3/run_tdnn.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```

Comment:
this should be only a symlink to a best-performing system in tuning.

Reply:
Yeah, I have turned it to be a symlink, please check it.